x-pack/test/functional/page_objects/gis_page.js

```diff
@@ -213,12 +213,16 @@ export function GisPageProvider({ getService, getPageObjects }) {
       return links.length;
     }
 
+    async isSetViewPopoverOpen() {
```

Comment:
These changes did not have anything to do with fixing functional tests. Just noticed when running these tests locally that waiting default 2500 for this check is painfully slow.